doc/source/index.rst

```diff
@@ -16,15 +17,13 @@ Indices and tables
 
 .. toctree::
    :hidden:
-   :maxdepth: 1
+   :maxdepth: 4
 
    about
+   faq
+   api
    install
-   data
-   devguide
+   examples
    gpu
-   releases
-   api
-   demo
-   faq
-   credits
+   devguide
+   zcredits
```

Comment:
z?

Reply:
I read somewhere that Sphinx processes files in alphabetic order, so this renaming prevents a bug where citations finalized before all of the in text substitutions are completed.

Reply:
Yes. You can add the following line in your ```conf.py``` file to use your ordering.
```
autodoc_member_order = 'bysource'
```